### PR TITLE
Update intro text

### PR DESCRIPTION
**Pull request by caseycesari**
## Overview

Updated to match the current copy on TNC's deployed version of the site.

## Demo

screen shot 2017-11-27 at 1 53 58 pm

## Testing Instructions

- Verify the text in the section highlighted in the above image matches the text on http://maps.coastalresilience.org/network/.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -121,7 +121,10 @@ <h2 class="color-white">
     <script type="text/template" id="regionListTemplate">
         <div class="location--list" id="region-location-list">
             <p>
-                At consectetur, reprehenderit ad distinctio sapiente obcaecati facilis dolor possimus officia incidunt consequatur in sed dignissimos neque.
+              Coastal Resilience is a program led by The Nature Conservancy to examine nature’s role in
+              reducing coastal flood risk. The program consists of an approach, a web mapping tool, and
+              a network of practitioners around the world supporting hazard mitigation and climate
+              adaptation planning.
```
Comment (kellyi) [inline src/index.html:127]: I'd bet that this is a typo in the other version but the http://maps.coastalresilience.org/network/ appears not to have a period after `adaptation planning`. Probably fine to leave it here, but thought I'd note the slight mismatch.

**Review by kellyi — APPROVED** [review]
+1

File: src/index.html (hunk 1 above)
Comment (caseycesari) [reply src/index.html:127]: Good catch. Going to leave it.

**Comment by caseycesari** [comment]
Thanks!

**Pull request merged**